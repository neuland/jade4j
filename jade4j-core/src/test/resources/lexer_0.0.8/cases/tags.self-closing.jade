
body
  foo
  foo(bar='baz')
  foo/
  foo(bar='baz')/
  foo /
  foo(bar='baz') /
  //- can have a single space after them
  img 
  //- can have lots of white space after them
  img    