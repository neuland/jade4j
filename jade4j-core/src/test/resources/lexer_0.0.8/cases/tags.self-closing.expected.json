{"type":"newline","line":2}
{"type":"tag","line":2,"val":"body","selfClosing":false}
{"type":"indent","line":3,"val":2}
{"type":"tag","line":3,"val":"foo","selfClosing":false}
{"type":"newline","line":4}
{"type":"tag","line":4,"val":"foo","selfClosing":false}
{"type":"attrs","line":4,"attrs":[{"name":"bar","val":"'baz'","escaped":true}]}
{"type":"newline","line":5}
{"type":"tag","line":5,"val":"foo","selfClosing":true}
{"type":"newline","line":6}
{"type":"tag","line":6,"val":"foo","selfClosing":false}
{"type":"attrs","line":6,"attrs":[{"name":"bar","val":"'baz'","escaped":true}],"selfClosing":true}
{"type":"newline","line":7}
{"type":"tag","line":7,"val":"foo","selfClosing":false}
{"type":"text","line":7,"val":"/"}
{"type":"newline","line":8}
{"type":"tag","line":8,"val":"foo","selfClosing":false}
{"type":"attrs","line":8,"attrs":[{"name":"bar","val":"'baz'","escaped":true}]}
{"type":"text","line":8,"val":"/"}
{"type":"newline","line":9}
{"type":"comment","line":9,"val":" can have a single space after them","buffer":false}
{"type":"newline","line":10}
{"type":"tag","line":10,"val":"img","selfClosing":false}
{"type":"text","line":10,"val":" "}
{"type":"newline","line":11}
{"type":"comment","line":11,"val":" can have lots of white space after them","buffer":false}
{"type":"newline","line":12}
{"type":"tag","line":12,"val":"img","selfClosing":false}
{"type":"text","line":12,"val":"   "}
{"type":"outdent","line":12}
{"type":"eos","line":12}